src/main/kotlin/ink/abb/pogo/scraper/tasks/LootOneNearbyPokestop.kt

```diff
@@ -31,7 +31,8 @@ class LootOneNearbyPokestop(val sortedPokestops: List<Pokestop>, val lootTimeout
 
         if (nearbyPokestops.size > 0) {
             val closest = nearbyPokestops.first()
-            Log.normal("Looting nearby pokestop ${closest.id}")
+            val pokestopName = nearbyPokestops.first().details.name
```

Comment:
@jabbink said that this adds one HTTPS request every looted log in #36 

You really want to add this added, maybe it's a good idea to add an option on the config file and a warning saying it will increase https requests.